x/leverage/keeper/oracle.go

```diff
@@ -14,11 +14,16 @@ import (
 // the base and display/symbol denominations for each exchange pair. E.g. it must
 // know about the UMEE/USD exchange rate along with the uumee base denomination
 // and the exponent.
-// This function will not return non-positive exchange rates, preferring
-// to error instead.
+// This function will not return non-positive exchange rates unless a token is
+// blacklisted, in which case it will return zero.
```

Comment:
```suggestion
// This function will only return positive exchange rates or errors, unless a
// token is blacklisted, in which case it will return zero.
```